File: src/algorithms/double-reversal.js
/**
 * Given an integer parameter num, reverses the integer to get reversed1, then
 * reverses reversed1 to get reversed2. Returns true if reversed2 equals num.
 * 
 * @param {number} num - A number equal than or greater than 0
 * @returns {boolean} - Returns true if num doubly reversed is still the same
 */
function doubleReversal (num) {
  let rev = Number(String(num).split('').reverse().join(''));
  rev = Number(String(rev).split('').reverse().join(''));
  return num === rev;
}

File: src/algorithms/double-reversal.py
def isSameAfterReversals(num):
    """Reversing an integer means to reverse all its digits. For example, 
    reversing 2021 gives 1202. Reversing 12300 gives 321 as the leading zeros 
    are not retained. Given an integer num, reverse num to get reversed1, then 
    reverse reversed1 to get reversed2. Return true if reversed2 equals num. 
    Otherwise return false.

    Args:
        num (int): A positive integer.

    Returns:
        Boolean: Returns True if num equals num reversed.
    """
    reverse = int("".join(reversed(str(num))))
    reverse = int("".join(reversed(str(reverse))))
    return num == reverse
